.env.example

```
@@ -304,6 +304,23 @@ ANTHROPIC_API_KEY=sk-ant-your-key-here
+#
+# Prefer structured tools on the server when possible, e.g.:
+#   llama-server -m model.gguf --jinja --port 8080
+#   (use the tool-call parser appropriate for your model, e.g. Qwen)
+#
+# CLAUDE_CODE_USE_OPENAI=1
+# OPENAI_BASE_URL=http://127.0.0.1:8080/v1
+# OPENAI_API_KEY=none
+# OPENAI_MODEL=qwen3.6:35b
+# OPENAI_SELF_HOSTED_TOOLS=1   # optional shell override; prefer profile flag
+
+
```

dotenv-linter (4.0.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)